browser/ui/toolbar/brave_vpn_menu_model.h

```diff
@@ -26,7 +27,7 @@ class BraveVPNMenuModel : public ui::SimpleMenuModel,
   void Build();
   bool IsBraveVPNButtonVisible() const;
 
-  Browser* browser_ = nullptr;
+  raw_ptr<Browser> browser_ = nullptr;
```

Comment:
What's the reason for using `raw_ptr` here and not in the VPN service pointer?

Reply:
@thypon I search all other places but can't find vpn service pointer is declared w/o `raw_ptr`.
Can you point out where it is?